specification/compute/resource-manager/Microsoft.Compute/stable/2019-04-01/skus.json

```diff
@@ -344,16 +349,12 @@
             "$ref": "#/definitions/ResourceSku"
           },
           "description": "The list of skus available for the subscription."
-        },
-        "nextLink": {
```

Comment:
Isn't this a breaking change?

Reply:
Hynho, we are not currently populating the nextlink for the get skus and it will be always null. This wont be a breaking change as we are not removing the property but indicating this property will be null unless in future we start populating the nextlink for this API